apps/transformer/lib/transform/move_field.ex

```diff
@@ -1,4 +1,15 @@
 defmodule Transform.MoveField do
+  @moduledoc """
+  `Transform.Step` impl to move or rename a field
+  in a dataset.
+
+  ## Configuration
+
+  * `from` - Required. Field to be moved. This can be a string or
+  path. Ex: `"foo"` or `["foo", "bar", "baz"]`
+  * `to` - Required. Where to move the field. This can be a string
+  or path. Ex: `"abc"` or `["abc", "xyz"]`
+  """
```

Comment:
Would it make sense to rename this step to `Transform.MvField` to make the correlation to the unix `mv` command clearer without having to read the docstring? Just thinking if I'm looking for "rename" functionality and just glancing at the module names I might skip this one.